go/webhook.go

```diff
@@ -77,12 +80,15 @@ func (wh *Webhook) Verify(payload []byte, headers http.Header) error {
 	return errNoMatchingSignature
 }
 
-func sign(key []byte, toSign string) []byte {
-	h := hmac.New(sha256.New, key)
+func (wh *Webhook) Sign(msgId, timestamp string, payload []byte) (string, error) {
```

Comment:
My only question is should we accept the time objects for the timestamp in each language or just a string?

Reply:
I'd change the timestamp parameter to int/long (rather than string). We care about it being seconds since epoch, so let's at least enforce that.